CONTRIBUTING.md

```diff
@@ -1,5 +1,74 @@
+  - [How Can I Contribute?](#how-can-i-contribute)
+    - [Reporting Bugs](#reporting-bugs)
+    - [Suggesting Enhancements](#suggesting-enhancements)
+    - [Code Contributions](#code-contributions)
+    - [Documentation](#documentation)
+  - [Getting Started](#getting-started)
+    - [Setting Up the Development Environment](#setting-up-the-development-environment)
+  - [Style Guide](#style-guide)
+  - [Review Process](#review-process)
+
+## How Can I Contribute?
+
+### Reporting Bugs
+
+Before submitting a bug report, please check if the issue has already been reported in the [issue tracker](https://github.com/nomtek/mistralai_client_dart/issues). If it hasn't, create a new issue with a clear title and description, including steps to reproduce the bug.
+
+### Suggesting Enhancements
+
+If you have an idea for an improvement or a new feature, we'd love to hear about it! Create a new issue in the [issue tracker](https://github.com/nomtek/mistralai_client_dart/issues) with your suggestion, and we can discuss it further.
+
+### Code Contributions
+
+We welcome contributions to the codebase. To contribute:
+
+1. [Fork the repository](https://help.github.com/en/github/getting-started-with-github/fork-a-repo).
+2. Create a new branch for your feature or bug fix.
+3. Make your changes and commit them with clear and concise messages.
+4. Push your changes to your fork.
+5. [Submit a pull request](https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/proposing-changes-to-your-work-with-pull-requests), explaining the changes you made.
+
+### Documentation
+
+Improving documentation is a valuable contribution. If you find areas that need clarification or have ideas for better explanations, please make the necessary updates.
+
+## Getting Started
+
+### Setting Up the Development Environment
+
+- Check [pubspec.yaml](https://github.com/nomtek/mistralai_client_dart/blob/main/pubspec.yaml) for any dependencies and versions required to build project.
+
+## Style Guide
+
+No particular style guide. We are using [very_good_analysis](https://pub.dev/packages/very_good_analysis) package to keep code consistent.
+
+## Review Process
+
+After submitting pull request few things will happen:
+
+1. **Reviewers Assigned:**
+   - The maintainers will be automatically notified of your pull request.
+   - They will assign one or more reviewers to assess your changes.
+
+2. **Address Feedback:**
+   - Respond promptly to any feedback or comments from reviewers.
+   - Make additional commits to address concerns raised during the review.
+
+3. **Continuous Integration (CI):**
+   - Automated tests will be run on your changes using our CI system.
+   - Ensure that all tests pass successfully.
+
+4. **Approval and Merging:**
+    - Once your changes are approved, a maintainer will merge your pull request.
+    - The changes will be reflected in the main branch.
+
+5. **Celebrate:**
+    - Congratulations! Your contribution has been successfully merged.
```

Comment:
Maybe we could also include those contributors in the release notes next to the changes they made? 😉

Reply:
Yeah, that's a good idea. I'll add it.

Reply:
Updated in https://github.com/nomtek/mistralai_client_dart/pull/24/commits/42354570f194dd5e88f3982f0b824e9e1bbca23b